src/Protocols.jl

```diff
@@ -251,8 +251,8 @@ end
 """
 function Error(msgid, error::T, stackframe) where {T}
     frames = [
-        (frame.file, frame.line, frame.func)
-        for frame in stackframe
+    (frame.file, frame.line, frame.func)
```

Comment:
This looks like a bug in Runic.

Reply:
Indeed!

Reply:
Fixed in https://github.com/fredrikekre/Runic.jl/commit/1184d2ba4395c82c49e6f4e8b1f08dbe3ab61ccd